src/related-manager.ts

```diff
@@ -132,7 +123,7 @@ class RelatedManager {
         internalIndex: index
       };
     });
-    this.dispatchEvent(RelatedEvent.RELATED_ENTRIES_CHANGED, this._entries);
+    this.dispatchEvent(new KalturaPlayer.core.FakeEvent(RelatedEvent.RELATED_ENTRIES_CHANGED, this._entries));
```

Comment:
I thought this event will still be fired from the player for external uses (analytics for example)